inbm/dispatcher-agent/dispatcher/dispatcher_class.py

```diff
@@ -310,6 +320,7 @@ def do_install(self, xml: str, schema_location: Optional[str] = None, job_id: st
         result: Result = Result()
         logger.debug("do_install")
         parsed_head = None
+        self._active_thread_manifest = xml
```

Comment:
review note: assumption in dispatcher is that there is only one active thread at a time